cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -88,6 +88,14 @@ function wait_for_istio_ready() {
     fi
 }
 
+function copy_istio_cni_conf_files() {
+  if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ] && [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
+    for nodeNum in $(seq -f "%02g" 1 $KUBEVIRT_NUM_NODES); do
+      $ssh node${nodeNum} -- sudo cp -uv /etc/cni/multus/net.d/*istio*.conf /etc/cni/net.d/
```

Comment:
Would creating a symlink be an option too? It would allow us to keep it generic and we would not need to wait for Istio to finish deployment

Reply:
@phoracek  it requires knowing the name of isito conf file that will be generated in order to create the link (though it won't likely to change), then it would be possible to create the link as part of `deploy_istio()` (before Isitio deployment ready), for example:
```
ISTIO_CONF_FILE_NAME="YYY-istio-cni.conf"
...
function deploy_istio() {
    if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ]; then
        if [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
        ...
        $ssh node${nodeNum} -- sudo ln -sr /etc/cni/multus/net.d/${ISTIO_CONF_FILE_NAME} /etc/cni/net.d/
...
```
WDYT?

Reply:
It wont work because Multus dosent look for net conf files recursively nor look into links, the conf file must be copied.

Reply:
i see, interesting, thanks for checking that